### ISSUE TITLE
[BitSail][Feature] Support generate constant value in FakeSource.

### ISSUE THREAD

**Issue by BlockLiu** [bytedance/bitsail#244]
# Please describe the feature you want

Subtask of #243

Support generate constant value in FakeSource connector.

**Comment by kyle-hawk** [comment]
@BlockLiu 
The document is about Legacy Fake connector, do we need to adjust the document of Fake Connector to v1 version? And add the description of this feature?

**Comment by BlockLiu** [comment]
Yeah, we'd better offer a document about existing connector and new features.
If you want to do that, here is a doc you may reference.

https://bytedance.github.io/bitsail/en/community/contribute.html#how-to-contribute-documents
